ui/src/app/store/device/selectors.ts

```diff
@@ -106,10 +106,62 @@ const eventErrorsForDevices = createSelector(
   }
 );
 
+/**
+ * Returns currently active device's system_id.
+ * @param {RootState} state - The redux state.
```

Comment:
We don't need these type annotations anymore

Reply:
Thanks, I'll remove them.